tests/qos/qos_sai_base.py

```diff
@@ -807,6 +807,10 @@ def dutConfig(
         dst_dut = get_src_dst_asic_and_duts['dst_dut']
         src_mgFacts = src_dut.get_extended_minigraph_facts(tbinfo)
         topo = tbinfo["topo"]["name"]
+        multi_asic_dut = False
+        if get_src_dst_asic_and_duts['src_dut'].sonichost.is_multi_asic \
+                and get_src_dst_asic_and_duts['dst_dut'].sonichost.is_multi_asic:
+            multi_asic_dut = True
```

Comment:
This still doesn't cover the case where you have `multi_dut` variant and only one of the `src_dut` or `dst_dut` is multi_asic. In that case you need to run the `sudo ip netns exec asic{}` versions of the commands on the multi-asic dut but not on the single-asic dut.

Reply:
@arista-nwolfe ,changes are made to cover both the single_asic  & multi_asic T2 broadcom-dnx devices for arp_populate test. Please review. 
Ping is needed make sure the Arp doesn't age out before qos test run completes.

Reply:
Thanks, new change looks good to me